packages/components/src/button/style.scss

```diff
@@ -9,7 +9,7 @@
 	background: none;
 	transition: box-shadow 0.1s linear;
 	@include reduce-motion("transition");
-	height: 36px;
+	height: $icon-button-size;
```

Comment:
We  should rename that variable (it's not just about icon) but we should do that separately.